pkgs/utils/test_utils/test_utls.go

```diff
@@ -34,33 +29,19 @@ type TestOperator struct {
 	Srv     *operator.Server
 }
 
-func ParseAsError(msg []byte) (error, error) {
-	sszerr := &wire.ErrSSZ{}
-	err := sszerr.UnmarshalSSZ(msg)
-	if err != nil {
-		return nil, err
-	}
-
-	return errors.New(string(sszerr.Error)), nil
-}
-
-func CreateTestOperatorFromFile(t *testing.T, id uint64, examplePath string, version string) *TestOperator {
+func CreateTestOperatorFromFile(t *testing.T, id uint64, examplePath, version string) *TestOperator {
 	err := logging.SetGlobalLogger("info", "capital", "console", nil)
 	require.NoError(t, err)
 	logger := zap.L().Named("operator-tests")
 	priv, err := crypto.EncryptedPrivateKey(examplePath+"operator"+fmt.Sprintf("%v", id)+"/encrypted_private_key.json", "12345678")
 	require.NoError(t, err)
 	r := chi.NewRouter()
-	db, err := kv.NewInMemory(logging.TestLogger(t), basedb.Options{
-		Reporting: true,
-		Ctx:       context.Background(),
-		Path:      t.TempDir(),
-	})
-	require.NoError(t, err)
 	operatorPubKey := priv.Public().(*rsa.PublicKey)
 	pkBytes, err := crypto.EncodePublicKey(operatorPubKey)
-	require.NoError(t, err)
-	swtch := operator.NewSwitch(priv, logger, db, []byte(version), pkBytes, id)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
more suitable to do this:
```suggestion
	require.NoError(t, err)
```